project.clj

```diff
@@ -5,13 +5,13 @@
             :url "http://iplantcollaborative.org/sites/default/files/iPLANT-LICENSE.txt"}
   :deploy-repositories [["releases" :clojars]
                         ["snapshots" :clojars]]
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [cheshire "5.6.3"]
-                 [metosin/compojure-api "1.1.8"]
+  :dependencies [[org.clojure/clojure "1.10.0"]
+                 [cheshire "5.8.1"]
+                 [metosin/compojure-api "1.1.12"]
                  [metosin/schema-tools "0.11.0"]
-                 [org.cyverse/clojure-commons "2.8.1"]
+                 [org.cyverse/clojure-commons "3.0.3-SNAPSHOT"]
```

Comment:
Should we cut a `3.0.3` release for `clojure-commons` and use that here?

Reply:
That sounds good to me. I figured I'd cut releases and update dependencies after merging these PRs.

Reply:
That works too 👌